src/libraries/System.IO.FileSystem.AccessControl/src/Resources/Strings.resx

```diff
@@ -122,6 +122,9 @@
   <data name="Arg_MustBeIdentityReferenceType" xml:space="preserve">
     <value>Type must be an IdentityReference, such as NTAccount or SecurityIdentifier.</value>
   </data>
+  <data name="Argument_InvalidAppendMode" xml:space="preserve">
+    <value>Append access can be requested only in write-only mode.</value>
```

Comment:
Added to match the .NET Framework case where `FileMode.Append` is used in combination with any read right.

https://referencesource.microsoft.com/#mscorlib/system/io/filestream.cs,739